.github/workflows/dev.yaml

```diff
@@ -0,0 +1,62 @@
+name: Release to dev
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  build:
+    runs-on: [ self-hosted, cere-network-large-workers ]
```

Comment:
Is it a good practice to use self-hosted runners in public repositories?

Reply:
No, but this is temporary change, while we not fully migrate to reusable workflows